flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml

```diff
@@ -3892,11 +3898,15 @@
             </UseTemplate>
             <UseTemplate Name="ASOBO_SAFETY_Push_Template">
                 <LEFT_SINGLE_CODE>
-                    0 (&gt;L:A32NX_MASTER_WARNING)
-                    1 (&gt;L:PUSH_AUTOPILOT_MASTERAWARN_R)
+                    (L:A32NX_OVERSPEED, Bool) 0 == (L:A32NX_LDG_NOT_DOWN, Bool) 0 == and if{
+                        0 (&gt;L:A32NX_MASTER_WARNING)
+                        1 (&gt;L:PUSH_AUTOPILOT_MASTERAWARN)
+                    } els{
+                        1 (&gt;L:PUSH_AUTOPILOT_MASTERAWARN)
+                    }
```

Comment:
**issue**: FWC logic programmed into the template.
**suggestion**: Don't include FWC logic in the template.

Reply:
Ok